crates/aptos/src/move_tool/mod.rs

```diff
@@ -540,6 +554,102 @@ impl CliCommand<&'static str> for ProvePackage {
     }
 }
 
+#[derive(Parser)]
+pub struct CoveragePackage {
```

Comment:
Please put a doc comment right above this, it'll add help to the CLI.  Something like:

```
/// Computes coverage for a package
///
/// Computes coverage on a previous unit test run for a package.  Coverage input must
/// first be built with `aptos move test --coverage`
```